src/clr-angular/forms/common/form.ts

```diff
@@ -28,4 +52,60 @@ export class ClrForm {
   markAsTouched() {
     this.markControlService.markAsTouched();
   }
+
+  @ContentChildren(ClrLabel, { descendants: true })
+  labels: QueryList<ClrLabel>;
+
+  @HostListener('submit')
+  onFormSubmit() {
+    this.markAsTouched();
+    if (isPlatformBrowser(this.platformId) && this.ariaLiveElement) {
+      this.invalidControls = Array.from(this.el.nativeElement.querySelectorAll('.ng-invalid'));
+      if (this.invalidControls.length > 0) {
+        this.invalidControls[0].focus();
+        this.updateAriaLive();
+      }
+    }
+  }
+
+  ngOnInit() {
+    this.createAriaLiveElement();
+  }
+
+  private updateAriaLive(): void {
+    if (this.invalidControls.length === 0) {
+      return;
+    }
+
+    this.ariaLiveElement.textContent =
```

Comment:
I find this awkward. This could be a loooong list. Was this the requirement or optional?

I think it would be more practical to state something like `There are # of invalid fields, you're now on the first error` since we're moving focus.

Reply:
This was a request during discussion with @clane. Chris, are you ok with the propsed text change from above?

Reply:
I think  "There are # of fields in error" is good enough. The focus movement will cause the label for the first field to read by the screen reader.  The live region should use aria-live="polite" so that that all announcements are heard and the content currently being read from the screen reader's cursor position is also still announced.